### DISCUSSION TITLE
Lesson 5: Unable to launch local RPC client.

### DISCUSSION THREAD

**Discussion by vicshi06** [smartcontractkit/full-blockchain-solidity-course-py#804] [Q&A] [answered]
Hi,

I followed everything in [this](https://medium.com/@cromewar/how-to-setup-windows-10-11-for-smart-contract-development-and-brownie-e7d8d13555b3) tutorial and installed WSL 2.

And I changed my ganache localhost to WSL. The projects are created in the WSL Home Foler, but I am still getting this message. 

```Python
Brownie v1.17.2 - Python development framework for Ethereum

Ethproject is the active project.

Launching 'ganache-cli --port 8545 --gasLimit 12000000 --accounts 10 --hardfork istanbul --mnemonic brownie'...
  File "brownie/_cli/__main__.py", line 64, in main
    importlib.import_module(f"brownie._cli.{cmd}").main()
  File "brownie/_cli/run.py", line 44, in main
    network.connect(CONFIG.argv["network"])
  File "brownie/network/main.py", line 50, in connect
    rpc.launch(active["cmd"], **active["cmd_settings"])
  File "brownie/network/rpc/__init__.py", line 93, in launch
    raise RPCProcessError(cmd, uri)
RPCProcessError: Unable to launch local RPC client.
Command: ganache-cli
URI: http://127.0.0.1:8545
```

Please help. I am be stuck here for a whole day. plz :(

**Comment by vicshi06** [comment, marked as answer]
I spent a whole day researching this and I finally resolved this.

You just need to downgrade your node into `16.13.2`. By using the following code [WSL2 ONLY!]:

```python
nvm uninstall <current version>

nvm install 16.13.2
```

Then you good!

**Reply by cromewar** [reply]
Hello @vicshi06 sorry for the delay, you are right is preferable to use LTS versions of node in order to work with web3, I will update my article with this info, Thanks

**Reply by freddie71010** [reply]
if it still doesn't work after installing version `16.13.2`, then make sure you run this extra command after installing:
```
nvm use 16.13.2
```
After you run this `use` command, try rerunning your script.

**Reply by cromewar** [reply]
Nice hint @freddie71010

**Comment by gnoparus** [comment]
It happens to me during lesson 13 after doing yarn stuff. Thanks for the answer.
